src/clr-angular/forms/styles/_file.clarity.scss

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('forms.control.file') {
+    .clr-control-container {
+        .clr-file-wrapper {
+            @include form-flatten-validate-text();
+            position: relative;
+            margin-top: $clr-forms-baseline;
+
+            .clr-file {
```

Comment:
`.clr-control-container .clr-file-wrapper .clr-file`

...to...

`.clr-file-wrapper .clr-file`